### PR TITLE
Doc general linux

### PR DESCRIPTION
**Pull request by sevagh**
rpm inspect:
```
shanssian:goat $ rpm -qp pkg/goat_0.9.0_x86_64.rpm -l
/lib/systemd/system/goat@.service
/usr/bin/goat
```

deb inspect:
```
root@92882e6ee3d7:/# dpkg -c goat_0.9.0_amd64.deb
drwxrwxr-x 0/0               0 2018-11-07 15:39 ./
drwxr-xr-x 0/0               0 2018-11-07 15:39 ./lib/
drwxr-xr-x 0/0               0 2018-11-07 15:39 ./lib/systemd/
drwxr-xr-x 0/0               0 2018-11-07 15:39 ./lib/systemd/system/
-rw-r--r-- 0/0             339 2018-11-07 15:39 ./lib/systemd/system/goat@.service
drwxr-xr-x 0/0               0 2018-11-07 15:39 ./usr/
drwxr-xr-x 0/0               0 2018-11-07 15:39 ./usr/share/
drwxr-xr-x 0/0               0 2018-11-07 15:39 ./usr/share/doc/
drwxr-xr-x 0/0               0 2018-11-07 15:39 ./usr/share/doc/goat/
-rw-r--r-- 0/0             146 2018-11-07 15:39 ./usr/share/doc/goat/changelog.gz
drwxrwxr-x 0/0               0 2018-11-07 15:39 ./usr/bin/
-rwxrwxr-x 0/0         6923304 2018-11-07 15:39 ./usr/bin/goat
```

@mvisonneau

### CODE REVIEW

**Comment by sevagh** [comment]
[Uploading goat_0.9.0_linux_amd64.zip…]()

**Comment by sevagh** [comment]
Apparently github won't let me attach a debian or RPM.

**Comment by sevagh** [comment]
This will solve:
https://github.com/sevagh/goat/issues/43
https://github.com/sevagh/goat/issues/42
https://github.com/sevagh/goat/issues/41

File: GNUmakefile [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+OS:=linux
+ARCH:=amd64
 GOAT_FILES?=$$(find . -name '*.go' | grep -v vendor)
 
 all: build
 
 build: deps
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -tags netgo -ldflags '-w -extldflags "-static" -X main.VERSION=$(VERSION)' -o bin/$(NAME)
-	strip bin/$(NAME)
+	CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build -a -tags netgo -ldflags '-w -extldflags "-static" -X main.VERSION=$(VERSION)' -o usr/bin/$(NAME)
```
Comment (mvisonneau) [inline GNUmakefile]: could even do `/usr/sbin` as this should be running as root only, wdyt ? 🤔

File: goat@.service [hunk 2]
```diff
@@ -9,7 +9,7 @@ ConditionPathExists=/usr/bin/goat
 Type=oneshot
 User=root
 Group=root
-ExecStart=/usr/bin/goat "%i" --log-level=info
+ExecStart=/usr/bin/goat "%i"
```
Comment (mvisonneau) [inline goat@.service]: not related to this MR, but it might be cool to be able to set flags either using a config file or env variables.

Comment (sevagh) [reply goat@.service]: My opinion is that there aren't enough flags for that to be worth the implementation cost

Comment (sevagh) [reply goat@.service]: We can revisit later

Comment (sevagh) [reply goat@.service]: But in the context of a systemd unit I can see why it's more useful to set an env var or config file than to change the unit constantly and do `daemon reload`...

**Comment by sevagh** [comment]
I added `GOAT_LOG_LEVEL` and `GOAT_TAG_PREFIX` as environment variables. `-version` and `-debug` don't make sense as env vars anyway.

**Pull request merged**